skyvern/forge/sdk/api/custom_credential_client.py

```
@@ -0,0 +1,272 @@
+        )
+
+        try:
+            response = await aiohttp_post(
+                url=url,
+                data=payload,
+                headers=headers,
+                raise_exception=True,
+            )
+
+            if not response:
+                raise HttpException(500, f"Empty response from custom credential API: {url}")
```

Ruff (0.14.7)

Abstract `raise` to an inner function

(TRY301)